microsoft-edge/dualengine/get-started.md

```diff
@@ -17,7 +17,7 @@ This article walks you through the steps to start using the DualEngine API.
 <!-- ====================================================================== -->
 ## Step 1: Get access to the DualEngine Limited Access Feature
 
-The DualEngine API is part of a Limited Access Feature. For more information or to request an unlock token, contact [Microsoft Support](https://support.serviceshub.microsoft.com/supportforbusiness/create?sapId=d15d3aa2-0512-7cb8-1df9-86221f5cbfde).
+The DualEngine API is part of a Limited Access Feature. For more information or to request an unlock token, contact [Microsoft Support](https://aka.ms/LAFAccessRequests).

```

Comment:
```suggestion
The DualEngine API is part of a Limited Access Feature. For more information or to request an unlock token, contact [Microsoft Support for business](https://aka.ms/LAFAccessRequests).
```
https://aka.ms/LAFAccessRequests
I presently end up at:
https://support.serviceshub.microsoft.com/supportforbusiness/onboarding?origin=/supportforbusiness/create --
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/ada64e88-b12d-4f35-9173-acb0ee70b8d5)

Reply:
We want to keep "Microsoft Support" here, the aka.ms redirection will soon point to another place.